### PR TITLE
Add some missing detail to badging article.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

**Comment by jpmedley** [comment]
@petele I found an email from Chris Mumford that listed something I forgot to address in the recent update of this article.

File: src/site/content/en/blog/badging-api/index.md [hunk 1]
```diff
@@ -132,10 +135,16 @@ For example:
 ```js
 // Set the badge
 const unreadCount = 24;
-navigator.setAppBadge(unreadCount);
+const setBadge = navigator.setAppBadge(unreadCount);
+setBadge.catch((error) => {
+  //Do something with the error.
+});
```
Comment (petele) [inline src/site/content/en/blog/badging-api/index.md]:
```suggestion
navigator.setAppBadge(unreadCount).catch((error) => {
  //Do something with the error.
});
```

File: src/site/content/en/blog/badging-api/index.md [hunk 2]
```diff
@@ -132,10 +135,16 @@ For example:
...
 
 // Clear the badge
-navigator.clearAppBadge();
+const clearBadge  = navigator.clearAppBadge();
+clearBadge.catch((error) => {
+  //Do something with the error.
+});
```
Comment (petele) [inline src/site/content/en/blog/badging-api/index.md]:
```suggestion
navigator.clearAppBadge().catch((error) => {
  // Do something with the error.
});
```

**Comment by jpmedley** [comment]
PTAL

(I debated about whether to do do the `catch()` blocks the way you did or the way I did. What's your heuristic for selecting for chosing?)

**Comment by petele** [comment]
> I debated about whether to do do the catch() blocks the way you did or the way I did. What's your heuristic for selecting for chosing?

My thought was that I'm never going to use this variable again, so why bother to create it. I think they're probably both just as good.

**Comment by kaycebasques** [comment]
@jpmedley I can't tell if you're going to make another change based on Pete's last comment or whether this is ready to merge so just LMK

**Comment by jpmedley** [comment]
This is ready for merge.

**Pull request merged**